tabbycat/users/groups.py

```diff
@@ -37,4 +73,36 @@ class TabDirector(BaseGroup):
 class TabAssistant(BaseGroup):
     # Permissions to match the Assistant interface
     name = _("Tabulation Assistant")
-    permissions = []
+    permissions = [
+        Permission.EDIT_ROUNDAVAILABILITIES_ADJ,
+        Permission.EDIT_ROUNDAVAILABILITIES_TEAM,
+        Permission.EDIT_ROUNDAVAILABILITIES_VENUE,
+        Permission.EDIT_ROOMCATEGORIES,
+        Permission.EDIT_ROOMCONSTRAINTS,
+        Permission.VIEW_ADMIN_DRAW,
+        Permission.EDIT_OLD_BALLOTSUBMISSIONS,
+        Permission.EDIT_BALLOTSUBMISSIONS,
+        Permission.ADD_BALLOTSUBMISSIONS,
+        Permission.MARK_BALLOTSUBMISSIONS,
+        Permission.MARK_OTHERS_BALLOTSUBMISSIONS,
+        Permission.VIEW_BALLOTSUBMISSION_GRAPH,
+        Permission.VIEW_FEEDBACK,
+        Permission.VIEW_FEEDBACK_OVERVIEW,
+        Permission.VIEW_FEEDBACK_UNSUBMITTED,
+        Permission.EDIT_FEEDBACK_CONFIRM,
+        Permission.EDIT_FEEDBACK_IGNORE,
+        Permission.ADD_FEEDBACK,
+        Permission.EDIT_BREAK_ELIGIBILITY,
+        Permission.EDIT_SPEAKER_CATEGORIES,
+        Permission.VIEW_PRIVATE_URLS,
+        Permission.GENERATE_PRIVATE_URLS,
+        Permission.SEND_PRIVATE_URLS,
+    ]
```

Comment:
<img width="1118" alt="Screenshot 2024-01-24 at 14 18 12" src="https://github.com/TabbycatDebate/tabbycat/assets/6293465/c2db1e7e-5252-49d9-8a03-be08d9d2d2a9">

I think there are way more permissions in this list than what they can actually do currently.